modyn/config/examples/example-pipeline.yaml

```diff
@@ -50,10 +50,9 @@ training:
   selection_strategy:
     name: NewDataStrategy
     maximum_keys_in_memory: 500000
-    config:
-      limit: -1
-      reset_after_trigger: True
-      storage_backend: database
+    limit: -1
```

Comment:
fyi. when running other pipeline configs from the repo, this also has to be adjusted. I didn't want to update all of these (mostly) duplicated configuration files in this PR.